week02_classification/seminar.ipynb

```diff
@@ -492,7 +492,7 @@
    "outputs": [],
    "source": [
     "model = SalaryPredictor()\n",
-    "batch = make_batch(data_train[:100])\n",
+    "batch = make_batch(data_train[:100], device=\"cpu\")\n",
```

Comment:
`make_batch()` has parameter `device` designed to put data to the right place. To avoid error, it is best to move the model to device, not data to CPU. Please change line 494 instead.

Reply:
Fixed